wa/workloads/googleslides/__init__.py

```diff
@@ -121,3 +121,21 @@ def init_resources(self, context):
         # Only accept certain presentation formats
         if os.path.splitext(self.test_file.lower())[1] not in ['.pptx']:
             raise ValidationError('{} must be a PPTX file'.format(self.test_file))
+
+    def deploy_assets(self, context):
+        super(GoogleSlides, self).deploy_assets(context)
+        #Move the test files to the download folder
+        d = self.target.working_directory
+        e = self.target.external_storage
+
+        orig_file_path = self.target.path.join(d, self.test_file)
+        new_file_path = self.target.path.join(e, 'Download')
```

Comment:
Just checking that you meant the 'Download' folder as opposed to the 'Downloads' folder? 
And also will this always be present on the device or should there be a check / creation of the folder?

Reply:
Downloads is correct. As far as I can tell, this is the default folder on all devices.